graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/fetchers/deployed/MutationFetcher.java

```diff
@@ -245,4 +251,43 @@ protected ExceptionWhileDataFetching toGraphqlError(
   protected SourceLocation getCurrentFieldLocation(DataFetchingEnvironment environment) {
     return environment.getMergedField().getSingleField().getSourceLocation();
   }
+
+  protected Function<List<MutationResult>, DataFetcherResult<Object>>
+      getDeleteOrUpdateResultBuilder(DataFetchingEnvironment environment) {
```

Comment:
Shared the code between update and delete per [this discussion](https://github.com/stargate/stargate/pull/1083#discussion_r671433875).